report_alternative_layout/models/ir_actions_report.py

```diff
@@ -32,10 +32,30 @@ def _get_partner(self, partner):
             return partner.commercial_partner_id
         return partner
 
+    def _get_bank_field(self, record):
+        """Get the field that links the record to the bank account.
+
+        We assume that there is usually just one field in the model with many2one
+        relationship to res.partner.bank. In case of an exception, this method should be
+        extended in the specific model to identify the correct field.
+        """
+        self.ensure_one()
+        return self.env["ir.model.fields"].search(
+            [
+                ("model", "=", record._name),
+                ("ttype", "=", "many2one"),
+                ("relation", "=", "res.partner.bank"),
+            ],
+            limit=1,
+        )
```

Comment:
@AungKoKoLin1997 Maybe this is better (haven't tested it)?
```suggestion
    @api.model
    def _get_bank_field(self, record):
        """Get the field that links the record to the bank account.

        We assume that there is usually just one field in the model with many2one
        relationship to res.partner.bank. In case of an exception, this method should be
        extended in the specific model to identify the correct field.
        """
        return record._fields.filtered(
            lambda x: x.ttype == "many2one" and x.relation == "res.partner.bank"
        )[:1]
```

Reply:
Forget it. It doesn't work as record._fields is a dict.